content/basics/use-ably.textile

```diff
@@ -0,0 +1,139 @@
+---
+title: How to use Ably
+meta_description: "A brief introduction on the options available to use with Ably"
+---
+
+Ably provides a number of APIs, SDKs, and adaptors and this page will briefly cover these options available to help you integrate with Ably in the way best suited for you and your business. 
+
+h2(#sdks). Ably SDKs with Realtime and REST Interfaces
+
+All official client library SDKs developed by Ably offer a consistent yet idiomatic API across every language. So regardless of your development environment or language, Ably keeps things simple for you by being consistent and predictable.
+
+The Ably platform also provides a protocol adapters service providing open protocol support and interoperability with a huge array of third party client libraries supporting every imaginable platform.
+
+Ably provides "a number of SDKS":https://ably.com/download in the most popular languages.
+
+h2(#realtime-api). Ably Realtime API
+
+The Ably Realtime client library establishes and maintains a persistent connection to Ably enabling low latency broadcasting and receiving of messages and presence state. All official client library APIs are consistent across every language offering pub/sub functionality, presence, token and basic authentication, active connection management, symmetric encryption and access to channel, presence and metrics history.
```

Comment:
Some of the features mentioned here are applicable to both the realtime and REST interfaces so should perhaps be in the opening section?